package.json

```diff
@@ -1768,6 +1768,9 @@
                     ".ipynb"
                 ]
             },
+            {
+                "id": "raw"
```

Comment:
We always supported raw cells, but were using `plaintext`,
Added `raw` so that its obvious in language picker whats `raw`, users won't know that `plaintext` is `raw cell`.